Makefile

```diff
@@ -305,9 +305,12 @@ ifeq ($(NATIVE), osx)
   CXXFLAGS += -mmacosx-version-min=$(OSX_MIN)
   LDFLAGS += -mmacosx-version-min=$(OSX_MIN)
   ifdef FRAMEWORK
-    FRAMEWORKSDIR := $(strip $(if $(shell [ -d $(HOME)/Library/Frameworks ] && echo 1), \
+    FRAMEWORKSDIR := $(shell echo $(FRAMEWORKSDIR))
```

Comment:
Uh, this is calling a shell in order to echo the make var `$(FRAMEWORKSDIR)` -- that's probably not what you want, as you're essentially assigning the variable to itself; it's a no-op, at best.